python/nano/src/bigdl/nano/deps/neural_compressor/onnx/pytorch/quantization.py

```diff
@@ -57,8 +57,4 @@ def collate_fn(batch):
         return model, calib_dataloader, metric
 
     def _post_execution(self, q_model):
-        # TODO Don't save, directly use q_model to create runtime
-        with TemporaryDirectory() as dir:
-            saved_onnx = Path(dir) / 'tmp.onnx'
-            q_model.save(saved_onnx)
-            return PytorchONNXRuntimeModel(str(saved_onnx))
```

Comment:
As shown in INC quantize with onnxruntime, we don't need to save the quantized model.